src/viur/core/modules/script.py

```diff
@@ -162,3 +163,40 @@ def update_path(self, skel):
             key = parent_skel["parententry"]
 
         skel["path"] = "/".join(path)
+
+    @exposed
+    def get_importable(self):
+        cuser = current.user.get()
+        if not cuser:
+            raise errors.Unauthorized()
+
+        current.request.get().response.headers["Content-Disposition"] = "attachment; filename=importable.zip"
+        current.request.get().response.headers["Content-Type"] = "application/zip"
```

Comment:
Do this stuff right before the return of the byte stream, especially when access control is done within.